src/entities/SearchForm/SearchForm.jsx

```diff
@@ -20,15 +20,24 @@ export const SearchForm = ({ initialValue = "", onSearch, placeholder }) => {
   };
 
   return (
-    <form className="search" onSubmit={() => onSearch(searchValue)}>
+    <form
+      className="search"
+      onSubmit={(event) => {
+        event.preventDefault();
+        onSearch(searchValue);
+      }}
+    >
       <input
         type="text"
         data-testid="search-input"
         className="search-input"
         value={searchValue}
         onChange={handleInputChange}
         onKeyDown={handleKeyPress}
-        onFocus={onSearch}
+        onFocus={(event) => {
```

Comment:
I would move this logic to a separate handler

Reply:
done